zask/ext/zerorpc/__init__.py

```diff
@@ -47,10 +47,12 @@
 CONFIG_CUSTOME_HEADER_MIDDLEWARE = 'header'
 ACCESS_LOG_MIDDLEWARE = 'access_log'
 REQUEST_CHAIN_MIDDLEWARE = 'uuid'
+REQUEST_EVENT_MIDDLEWARE = 'event'
 DEFAULT_MIDDLEWARES = [
     CONFIG_CUSTOME_HEADER_MIDDLEWARE,
     REQUEST_CHAIN_MIDDLEWARE,
-    ACCESS_LOG_MIDDLEWARE
+    ACCESS_LOG_MIDDLEWARE,
+    REQUEST_EVENT_MIDDLEWARE
```

Comment:
Ok. I was thinking of asking if we could arrange them alphabetically but not important at this point if we are not sure of the consequence on behavior.